HPXMLtoOpenStudio/measure.rb

```diff
@@ -2370,7 +2370,10 @@ def self.add_airflow(runner, model, weather, spaces)
     open_window_area = window_area * @frac_windows_operable * 0.5 * 0.2 # Assume A) 50% of the area of an operable window can be open, and B) 20% of openable window area is actually open
     site_type = @hpxml.site.site_type
     shelter_coef = @hpxml.site.shelter_coefficient
-    has_flue_chimney = false # FUTURE: Expose as HPXML input
+    if @hpxml.building_construction.has_flue_or_chimney.nil?
+      @hpxml.building_construction.has_flue_or_chimney = false
+    end
```

Comment:
This logic should go in hpxml_defaults.rb and then you should add tests in test_defaults.rb